fractal-puppy/Engine
# Feature/hotkeys n shortcuts

Also refactured GameLoop to work following a state system with separated classes.

To test everything:

Load Menuscene

- Check that dying, winning and everything in the Gameloop aspect keeps working the same way
(pro tip, make the player faster in the script)
- When in the graveyard level keys V,B and N will open the submenus. Escape closes all of them.

(Escape no longer closes the engine when in play mode!)
